### PR TITLE
Refactored the Terraform config to make our naming convention explicit

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

File: infrastructure/env/production/tfvars.json [hunk 1]
```diff
@@ -3,26 +3,10 @@
   "environment_short": "prod",
   "location": "West Europe",
   "cosmosdb_failover_location": "North Europe",
-  "azurerm_resource_group": "agid-rg-prod",
```
Comment (gunzip) [inline infrastructure/env/production/tfvars.json]: i feat that some typescript task use this values (see lib/config.ts). have you checked that it's not the case ?

Comment (cloudify) [reply infrastructure/env/production/tfvars.json]: I guess [this list](https://github.com/teamdigitale/digital-citizenship/blob/master/lib/config.ts#L59) needs to be reviewed as we moved many tasks to TF provisioners?

Comment (gunzip) [reply infrastructure/env/production/tfvars.json]: for example the api_management task read all the values from tsconfig.json to replace them in the API management templates

Comment (gunzip) [reply infrastructure/env/production/tfvars.json]: we can close this PR in favor of #54 which resolves the issue

Comment (cloudify) [reply infrastructure/env/production/tfvars.json]: great, thanks

**Comment by cloudify** [comment]
See PR #54

**Pull request closed without merging**